spec/factories/permission_templates.rb

```diff
@@ -45,13 +45,21 @@
         workflow = create(:workflow, active: true, permission_template: permission_template)
         create(:workflow_action, workflow: workflow) # Need to create a single action that can be taken
       end
+      if evaluator.depositor.present?
+        create(:permission_template_access,
+               :manage,
+               permission_template: permission_template,
+               agent_type: 'user',
+               agent_id: evaluator.depositor.user_key)
+      end
     end
 
     transient do
       with_admin_set false
       with_collection false
       with_workflows false
       with_active_workflow false
+      depositor nil
```

Comment:
I believe we call this attribute `user` on all the other factories.  Can we change the name for consistency?